examples/src/main/scala/io/finch/middleware/Main.scala

```diff
@@ -52,6 +53,14 @@ object Main extends App with Endpoint.Module[IO] {
   val filters = Function.chain(Seq(stats, logging, auth))
   val compiled = filters(Bootstrap.serve[Text.Plain](helloWorld).compile)
 
-  Await.ready(Http.server.serve(":8081", Endpoint.toService(compiled)))
+  def serve(service: Service[Request, Response]): Resource[IO, ListeningServer] =
+    Resource.make(IO(Http.server.serve(":8081", service))) { server =>
```

Comment:
An idea for another PR: maybe we should make this function a utility and provide our wrapper around IOApp.

Reply:
Definitely! Is there any place to add feature request?

Reply:
Maybe just file a Github issue for now?